js_modules/dagster-ui/packages/ui-core/src/instance/InstanceConfig.tsx

```diff
@@ -18,29 +18,31 @@ import {createGlobalStyle} from 'styled-components';
 import {InstancePageContext} from './InstancePageContext';
 import {InstanceTabs} from './InstanceTabs';
 import {InstanceConfigQuery, InstanceConfigQueryVariables} from './types/InstanceConfig.types';
-import {FIFTEEN_SECONDS, useQueryRefreshAtInterval} from '../app/QueryRefresh';
+import {
+  FIFTEEN_SECONDS,
+  QueryRefreshCountdown,
+  useQueryRefreshAtInterval,
+} from '../app/QueryRefresh';
 import {useTrackPageView} from '../app/analytics';
 import {useDocumentTitle} from '../hooks/useDocumentTitle';
 
 const InstanceConfigStyle = createGlobalStyle`
-  .CodeMirror.cm-s-instance-config {
+  .CodeMirror.cm-s-instance-config.cm-s-instance-config {
```

Comment:
Hmm the classnames are identical here? Not sure what this second one does in this context but maybe I've just never tried :-o